articles/cosmos-db/mongodb/mongodb-time-to-live.md

```diff
@@ -22,16 +22,10 @@ MongoShell example:
 
 ```
 globaldb:PRIMARY> db.coll.createIndex({"_ts":1}, {expireAfterSeconds: 10})
-{
-        "_t" : "CreateIndexesResponse",
-        "ok" : 1,
-        "createdCollectionAutomatically" : true,
-        "numIndexesBefore" : 1,
-        "numIndexesAfter" : 4
-}
 ```
+The command in the above example will create an index with TTL functionality. 
```

Comment:
```suggestion
The command in the above example will create an index with TTL functionality. 

The output of the command includes various metadata:

```output
{
        "_t" : "CreateIndexesResponse",
        "ok" : 1,
        "createdCollectionAutomatically" : true,
        "numIndexesBefore" : 1,
        "numIndexesAfter" : 4
}
```
```

There's still value to including the output, even if it's in a new code block